packages/@react-aria/calendar/test/useCalendar.test.js

```diff
@@ -17,15 +17,10 @@ import React from 'react';
 
 describe('useCalendar', () => {
   beforeAll(() => {
-    jest.useFakeTimers();
+    jest.useFakeTimers('legacy');
     jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => cb());
   });
 
-  afterAll(() => {
-    jest.useRealTimers();
-    window.requestAnimationFrame.mockRestore();
```

Comment:
Why the removal of the mockRestore and in the other files? Are we making the assumption now that the same requestAnimationFrame mock should be used in every test?

Reply:
suites are independent, it shouldn't cross over between, so if it was in an afterAll at the top, i just went ahead and removed it since jest changed such that this is no longer a valid way to restore anyways, instead i could change it to 

```
let rafSpy;
rafSpy = jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => cb());
rafSpy.mockRestore();
```

but again, in a top level afterAll, it shouldn't matter because everything after a suite is restored

Reply:
Oh snap, ok thats fine w/ me then, didn't realize that the mocks didn't carry over from test to test anymore